big_tests/tests/mim_c2s_SUITE.erl

```diff
@@ -119,6 +121,16 @@ message_sent_to_malformed_jid_results_in_error(Config) ->
         escalus_assert:is_chat_message(<<"Hi!">>, escalus_client:wait_for_stanza(Bob))
     end).
 
+invalid_resource_fails_to_log(Config) ->
```

Comment:
Wasn't this already done in a merged PR? Maybe rebase?

Reply:
I think I did and yet dunno why this shows up here, just rebased again.